rewardsPointsSystem.py

```diff
@@ -31,71 +25,57 @@
 		- Customer 1 purchased 2 bananas and 1 apple; he used 100 rewards points
 		- Customer 2 purchased 1 banana and 1 apple; she used 0 rewards points
 		- Customer 1 purchased 1 banana; he used 0 rewards points
-
 """
+from collections import defaultdict
 
 class Item:
-  def __init__(self, itemId, item_price):
-    self.itemId = itemId
-    self.item_price = item_price
-
-from collections import defaultdict    
+    """
+    A class that represents an item, along with its ID and price.
+    """
+  def __init__(self, item_id, item_price):
+    self.id = item_id
+    self.price = item_price
 
 class RewardsSystem:
-  REWARDS_RATIO_BELOW = 18
-  REWARDS_CUTOFF = 250
```

Comment:
How did you implement this condition: "customers earn 1 reward point for every 18 dollars spent, unless they spend more than $250 per day, in which case they earn 1 reward point for every 17 dollars spent."?

Reply:
Good job catching the redundant code blocks! Also, it could help to comment on the 'simplified' logic since shortening the code also makes it less intuitive at times.